backend/tests/v5/e2e/routes/teamspaces/projects/models/common/tickets.test.js

```diff
@@ -606,6 +606,7 @@ const testUpdateTicket = () => {
 	describe('Update ticket', () => {
 		const { users, teamspace, project, con, fed } = generateBasicData();
 		const requiredPropName = ServiceHelper.generateRandomString();
+		const immutableProperty = ServiceHelper.generateRandomString();
```

Comment:
Can we add a test for (not sure where abouts in the file, i'm just tagging it here, doesn't mean we should do it here):

- A template with an immutable property that has a default value
- Create a ticket without filling in the property
- try to update the timmutable property - should fail (as it should've taken the default value)